1223-curator_worklist/my_worklist.feature

```diff
@@ -0,0 +1,190 @@
+  Scenario Outline: Curator views all Requests of a type
+    When Curator click on Requests of type "<Type>"
+    Then Curator see a list of Requests displayed with:
+      | Request status            |
+      | Registration title        |
+      | Submitter name            |
+      | Request Submitter Date    |
+      | Beginning of last message |
+      | Owner name                |
+    And they see that each Request can be opened
+    Examples:
+      | Type      |
+      | Approval  |
+			# Må handtere at Editor kan tillate alle å publisere - kva skjer da med Approvals?
+      | Support   |
+      | DOI       |
+      | Ownership |
+
+  @needJiraTag
+  Scenario: Curator open a unassigned Request
+    When the Curator open a unassigned Request
+    Then the Curator is assigned the Request
+    And the Request Status is set to "Active"
+
+  @needJiraTag
+  Scenario: Curator unassigns a Request
+    When the Curator selects "Mark request unread"
+    Then the Request Status is set to "New"
+    And the Request is unassigned the Curator
+
+  @1252
+  @updated
+  Scenario: Curator open a assigned Request
+    When the Curator selects a Request
+    Then the Request is expanded
+    And the assigned Curator is viewed
+    And previous messages are displayed chronologically with:
+      | Submitter name            |
+      | Submitter Date and Time   |
+      | The full message          |
+    And the Curator can reply to a message
+    And the Curator can open the Requests Resource
+    And the Curator can change the Status of the Request
+
+  @needJiraTag
+  Scenario Outline: Curator open the Request's Resource
+    When the Curator opens the Requests Resource
+    Then the Landing Page of the Resource is viewed
+    And the Resource "<Type>" Button "<Action>" is visible
+    Examples:
+      | Type      | Action       |
+      | Approval  | Publish      |
+      | Approval  | Decline      |
+      | DOI       | Mint DOI     |
+      | DOI       | Decline      |
+      | Ownership | Change owner |
+      | Ownership | Decline      |
+
+ @needJiraTag
+  Scenario: User gets an answer to a Support Request
+    When the Curator sends an answer of type "Support"
+    Then the Request status is set to "Answered"
+    And the User can read the answer in My Messages
+
+  @needJiraTag
+  Scenario: User gets an answer to a Request
+    When the Curator writes an answer
+    And sends it to the User
+    And the Request Type is:
+      | Approval  |
+      | DOI       |
+      | Ownership |
+    Then the User can see the answer in My Messages
+
+  @needJiraTag
+  Scenario Outline: Curator change Status on a Request
+    When Curator selects a new status "<Status>" on a Request
+    Then the status of the Request is set to "<Status>"
+    Examples:
+      | Status   |
+      | Archived |
+      | Deleted  |
+
+  @needJiraTag
+  Scenario: Curator receives assignment of responses to requests they have previously responded to
```

Comment:
Curator scope contains responses to requests to which they have previously responded (maybe?)